.github/workflows/pull-requests.yaml

```
@@ -153,5 +153,8 @@ jobs:
       - name: Set sandbox ID
         run: echo "SANDBOX_NAME=cozy-e2e-sandbox-$(echo "${GITHUB_REPOSITORY}:${GITHUB_WORKFLOW}:${GITHUB_REF}" | sha256sum | cut -c1-10)" >> $GITHUB_ENV
 
-      - name: E2E Apps
+      - name: List images used
+        run: make -C packages/core/testing SANDBOX_NAME=$SANDBOX_NAME list-images
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:44: Double quote to prevent globbing and word splitting

(shellcheck)